src/Microsoft.ML.Data/Transforms/ConversionsCatalog.cs

```diff
@@ -35,5 +36,23 @@ public static class HashingEstimatorCatalog
         public static HashEstimator Hash(this TransformsCatalog.Conversions catalog, params HashTransformer.ColumnInfo[] columns)
             => new HashEstimator(CatalogUtils.GetEnvironment(catalog), columns);
 
+        /// <summary>
+        /// Changes column type of the input column.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="inputColumn">Name of the input column.</param>
+        /// <param name="outputColumn">Name of the column to be transformed. If this is null '<paramref name="inputColumn"/>' will be used.</param>
```

Comment:
Name of the column resulting from the transformation of <paramref name="inputColumn">.